flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_Transition.js

```diff
@@ -0,0 +1,318 @@
+class A32NX_Transition {
+    init() {
+        console.log('A32NX_TransitionAltitude init');
+        this.currentDeparture = "";
+        this.currentArrival = "";
+        this.offline = false;
+        this.serverStatus();
+    }
+
+    update(_deltaTime, _core) {
+    }
+
+    serverStatus() {
+        if (!NXApi.hasTelexConnection()) {
+            this.offline = true;
+        }
+    }
```

Comment:
Again, I am aware this is a draft but I'll just leave a few reviews. Firstly, I recommend changing the variable to check if we are online, not if we aren't. Secondly, instead of having the `serverStatus` function, which you only use once, you can just set the value of `online` when it's declared.

```suggestion
        this.online= NXApi.hasTelexConnection();
    }

    update(_deltaTime, _core) {
    }
```